sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/specialized/ShareLeaseAsyncClient.java

```diff
@@ -193,17 +242,46 @@ public String getLeaseId() {
     @ServiceMethod(returns = ReturnType.SINGLE)
     public Mono<Response<Void>> breakLeaseWithResponse() {
         try {
-            return withContext(this::breakLeaseWithResponse);
+            return breakLeaseWithResponse(new ShareBreakLeaseOptions());
+        } catch (RuntimeException ex) {
+            return monoError(logger, ex);
+        }
+    }
+
+    /**
+     * Breaks the previously acquired lease, if it exists. Leases will break immediately.
```

Comment:
This is only true of files, right? Shares will respect the break period

Reply:
yep, broke it up into behavior for shares and files.